### PR TITLE
Alpine 3.4 with php5.6 setup

### PR DESCRIPTION
**Pull request by amritgc**
We will be using this images for wercker build process, Later on we will use on our development environment

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	php5-mcrypt \
+	php5-opcache \
+	php5-sqlite3 \
+	php5-pdo_sqlite \
+	php5-ctype \
+	php5-zlib \
+	php5-curl \
+	php5-gd \
+	php5-xml \
+	php5-dom ;
```
Comment (geshan) [inline Dockerfile]: might need python for bitirx notification.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer; \
+    chmod +x /usr/local/bin/composer;
+RUN composer global require hirak/prestissimo
+RUN rm -rf /var/cache/apk/*
```
Comment (geshan) [inline Dockerfile]: I would put this in line 24 to avoid one extra layer.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+RUN mkdir -p /var/www
+WORKDIR /var/www
+COPY . /var/www
+VOLUME /var/www
```
Comment (geshan) [inline Dockerfile]: I would just add the volume on the docker-compose level not here as its better to make the container immutable.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+CMD ["/bin/sh"]
+EXPOSE 80 9000
```
Comment (geshan) [inline Dockerfile]: not sure of the ports too, might do it on the docker-compose level.

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+ENTRYPOINT ["/bin/sh", "-c"]
```
Comment (geshan) [inline Dockerfile:33]: add a docker-compose file too so that the user just does `docker-compose bulid` and ready to go.

File: Dockerfile (hunk 1 above)
Comment (geshan) [reply Dockerfile]: Do you want to deal with xdebug on this level?

File: Dockerfile (hunk 5 above)
Comment (geshan) [reply Dockerfile:33]: what about php.ini or any other custom config files?

File: Dockerfile (hunk 1 above)
Comment (amritgc) [reply Dockerfile]: No, not here that will be on added on wercker build

File: Dockerfile (hunk 3 above)
Comment (amritgc) [reply Dockerfile]: Okay

File: Dockerfile (hunk 4 above)
Comment (amritgc) [reply Dockerfile]: yup, I have already realized that

File: Dockerfile (hunk 5 above)
Comment (amritgc) [reply Dockerfile:33]: I have only focused for wercker build right now i will add necessary file as we need them

**Pull request merged**